python/ray/serve/controller.py

```diff
@@ -332,7 +348,8 @@ async def deploy(self,
                 version=version,
                 backend_config=backend_config,
                 replica_config=replica_config,
-                deployer_job_id=deployer_job_id)
+                deployer_job_id=deployer_job_id,
+                start_time_ms=int(time.time() * 1000))
```

Comment:
should this be clocked at beginning of deployment or when the all replicas become available? @edoakes